meson.build

```diff
@@ -377,7 +377,9 @@ srcfiles_unittest = files(
   'test/redfish-core/lib/log_services_dump_test.cpp',
   'test/redfish-core/lib/service_root_test.cpp',
   'test/redfish-core/lib/thermal_subsystem_test.cpp',
-  'test/redfish-core/lib/power_subsystem_test.cpp',
+  # Downstream enhancements require D-Bus calls for PowerSubsystem.  Those calls
+  # fail with a SIGSEGV in the gtest environment.
+  # 'test/redfish-core/lib/power_subsystem_test.cpp',
```

Comment:
Can you just remove this file?

Reply:
Sure. I thought maybe if we rebase again and they improve the upstream gtest environment, it would be easier to just uncomment this. But if you'd prefer removing the file I can do that.

Reply:
Done